modules/assim.batch/R/pda.emulator.R

```diff
@@ -209,8 +209,11 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
     
       ## read model outputs    
       for (i in seq_len(settings$assim.batch$n.knot)) {
-        model.out[[i]] <- pda.get.model.output(settings, run.ids[i], bety, inputs)
+        align.return <- pda.get.model.output(settings, run.ids[i], bety, inputs)
+        model.out[[i]] <- align.return$model.out
       }
+      inputs <- align.return$inputs
```

Comment:
Ahh, I think I was misunderstanding and thought we were looping over data not knots. But now that you mention it it would fail if the final run had failed.